upgrade_deadlines/repair.py

```diff
@@ -6,39 +6,59 @@
 from time import sleep
 
 import dryscrape
+import requests
 from dateutil.parser import parse
-from edx_rest_api_client.client import EdxRestApiClient
-
 
 logger = logging.getLogger(__name__)
 
 Run = namedtuple('Run', ['key', 'upgrade_deadline', 'end'])
 
-OIDC_URL = os.environ.get('OIDC_URL', 'https://courses.edx.org/oauth2')
-OIDC_KEY = os.environ.get('OIDC_KEY')
-OIDC_SECRET = os.environ.get('OIDC_SECRET')
+OAUTH_ACCESS_TOKEN_URL = os.environ.get('OAUTH_ACCESS_TOKEN_URL', 'https://courses.edx.org/oauth2')
+OAUTH_KEY = os.environ.get('OAUTH_KEY')
+OAUTH_SECRET = os.environ.get('OAUTH_SECRET')
 
 DISCOVERY_API_URL = os.environ.get('DISCOVERY_API_URL', 'https://prod-edx-discovery.edx.org/api/v1/')
 ECOMMERCE_URL = os.environ.get('ECOMMERCE_URL', 'https://ecommerce.edx.org')
 
 LMS_EMAIL = os.environ.get('LMS_EMAIL')
 LMS_PASSWORD = os.environ.get('LMS_PASSWORD')
+USER_AGENT = os.environ.get('API_CLIENT_NAME', 'ecommerce-scripts')
+
+
+def get_access_token():
+    response = requests.post(
+        f'{OAUTH_ACCESS_TOKEN_URL}/access_token',
+        data={
+            'grant_type': 'client_credentials',
+            'client_id': OAUTH_KEY,
+            'client_secret': OAUTH_SECRET,
+            'token_type': 'jwt',
+        },
+        headers={
+            'User-Agent': USER_AGENT,
+        },
+        timeout=(3.1, 5)
+    )
+    data = response.json()
+    return data['access_token']
```

Comment:
It seems that potentials exceptions for making requests as well as for taking a value for the key are not handled, should it be added somewhere?

Reply:
added handling exceptions